engine/src/main/java/org/terasology/rendering/dag/nodes/PrePostCompositeNode.java

```diff
@@ -190,6 +154,8 @@ public void process() {
             prePostMaterial.setFloat4("skyInscatteringSettingsFrag", skyInscatteringSettingsFrag, true);
         }
 
+        // TODO: We never set the "fogWorldPosition" uniform in prePostComposite_frag.glsl . Either use it, or remove it.
+
```

Comment:
With that being said, are you fine with that comment? It serves as a reminder that this feature is broken :D